Misc/NEWS.d/next/Tests/2019-04-02-09-25-31.bpo-36503.KrA0WF.rst

```diff
@@ -0,0 +1,2 @@
+Remove references to "aix3" and "aix4"
+patch by M Felt
```

Comment:
```suggestion
Patch by M. Felt.
```

Reply:
@aixtools Please add your name in `Misc/ACKS` and use same notation in NEWS entry.